src/main/kotlin/com/manimdsl/stylesheet/Stylesheet.kt

```diff
@@ -77,14 +78,28 @@ class Stylesheet(private val stylesheetPath: String?, private val symbolTableVis
         val dataStructureStyle =
             stylesheet.dataStructures.getOrDefault(value.toString(), StyleProperties())
         val style = stylesheet.variables.getOrDefault(identifier, dataStructureStyle)
-        return style merge dataStructureStyle
+
+        val newStyle = style merge dataStructureStyle
+        val animatedStyle = getAnimatedStyle(identifier, value)
+        if (animatedStyle != null) {
+            if (animatedStyle.borderColor != null && newStyle.borderColor == null) {
```

Comment:
There was an issue with animations not switching back to the default WHITE when there was no textColor specified anywhere in the stylesheet. They would just stay the animation colour